backend/Makefile

```diff
@@ -75,11 +75,11 @@ db/local/load-data:
 
 db/local/load-schema:
     # Imports the corpora_dev.sqlc schema (schema ONLY) into the corpora_test database
-	# Usage: make db/local/load-schema INFILE=<file>
+    # Usage: make db/local/load-schema INFILE=<file>
     $(eval DB_PW = $(shell aws secretsmanager get-secret-value --secret-id corpora/backend/test/database --region us-west-2 | jq -r '.SecretString | match(":([^:]*)@").captures[0].string'))
-	PGPASSWORD=${DB_PW} pg_restore --schema-only --clean --no-owner --host 0.0.0.0 --dbname corpora $(INFILE)
-	# Also import alembic schema version
-	PGPASSWORD=${DB_PW} pg_restore --data-only --table=alembic_version --no-owner --host 0.0.0.0 --dbname corpora $(INFILE)
+	PGPASSWORD=${DB_PW} pg_restore --schema-only --schema=persistence-schema --clean --no-owner --host 0.0.0.0 --username corpora --dbname corpora $(INFILE)
```

Comment:
For now don't we want both schemas, until we depreciate corpora_orm.py

Reply:
I wasn't able to get it to work without adding this arg, I kept getting strange pg_restore errors. For example:
```
pg_restore: error: could not execute query: ERROR:  extension "aws_s3" does not exist
Command was: DROP EXTENSION aws_s3;
pg_restore: error: could not execute query: ERROR:  extension "aws_commons" does not exist
Command was: DROP EXTENSION aws_commons;
```
I figured limiting this to the persistence_schema is fine, given that this command is only used to load the schema into the test DB to detect diffs after a test migration. By the time this is in main, we won't have plans to support schema changes in corpora_orm anymore. And if we have to revert for whatever reason, this would be reverted as well.